js-exercises/update-object/updateObject.js

```diff
@@ -0,0 +1,17 @@
+
+function updateObject(...args) {
+  let indexToReplace = args[0];
+  let strToBeReplaced = args[1];
+  let arrInput = args[2];
+
+  if(indexToReplace === -1) {
+    arrInput[arrInput.length - 1] = strToBeReplaced;
+  } else {
+    arrInput[indexToReplace] = strToBeReplaced;
+  }
```

Comment:
This logic could have handled more negative numbers than -1. Like we could set the index range to -(string length) to (string length). If we get anything outside this, we could throw an exception